health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AddOrEditMapping.js

```
@@ -0,0 +1,190 @@
+                            selected={selectedLevel}
+                            disabled={false}
+                            isMandatory={true}
+                            option={boundaryHierarchy}
+                            select={(value) => {
+                                setSelectedLevel(value);
+                                setSelectedBoundary(null);
+                            }}
+                            optionKey="boundaryType"
+                            t={t}
+                        />
+                        <div style={{height:"1rem"}}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)